fastlane/lib/fastlane/actions/create_pull_request.rb

```diff
@@ -6,34 +6,35 @@ module SharedValues
 
     class CreatePullRequestAction < Action
       def self.run(params)
-        require 'excon'
-        require 'base64'
-
         UI.message("Creating new pull request from '#{params[:head]}' to branch '#{params[:base]}' of '#{params[:repo]}'")
 
-        url = "#{params[:api_url]}/repos/#{params[:repo]}/pulls"
-        headers = { 'User-Agent' => 'fastlane-create_pull_request' }
-        headers['Authorization'] = "Basic #{Base64.strict_encode64(params[:api_token])}" if params[:api_token]
-
-        data = {
+        request_body = {
```

Comment:
agreed! from scanning the code it looks like a good pattern would be `payload` for request json, as `json` is used as the response parameter in most other places. I've changed all other areas in this PR to match too.